assets/css/screenplay.scss

```diff
@@ -74,7 +65,24 @@ input[type="button"] {
   letter-spacing: 0.5px;
 }
 
+.h2 {
+  font-weight: 700;
+  font-size: 28px;
+}
```

Comment:
just popping in to note that Mary says here H2 is [actually larger](https://www.figma.com/file/FKWRVrSdoMjZOLorhuykvc?node-id=1760:58899&mode=dev#645365932)